packs/data/equipment.db/energy-mutagen-major.json

```diff
@@ -29,7 +29,7 @@
         },
         "containerId": null,
         "description": {
-            "value": "<p><strong>Activate</strong> <span class=\"pf2-icon\">A</span> Interact</p>\n<hr />\n<p>When created, this mutigen is attuned to one of four energy types: acid, cold, electricity, or fire. When consumed, the mutagen suffuses your body with energy that spills out whenever you attack. At higher levels, it can even grant you the ability to unleash the energy in controlled bursts.</p>\n<p><strong>Benefit</strong> You gain resistance 20 to the attuned energy type. Whenever you score a hit with a melee weapon, add [[/r 2d6]] damage to the attuned energy type. In addition, you can end the benefits of this mutagen to unleash a @Template[type:cone|distance:30] of energy that deals [[/r 3d6]] damage of the attuned type for every full 10 minutes of duration remaining (@Check[type:reflex|dc:32|basic:true|name:Major Energy Mutagen] save).</p>\n<p><strong>Drawback</strong> You gain weakness 5 to the other three energy types.</p>\n<p><strong>Duration</strong> 1 hour.</p>\n<p>@Compendium[pf2e.equipment-effects.Effect: Energy Mutagen (Major)]{Effect: Energy Mutagen (Major)}</p>"
+            "value": "<p><strong>Activate</strong> <span class=\"pf2-icon\">A</span> Interact</p>\n<hr />\n<p>When created, this mutagen is attuned to one of four energy types: acid, cold, electricity, or fire. When consumed, the mutagen suffuses your body with energy that spills out whenever you attack. At higher levels, it can even grant you the ability to unleash the energy in controlled bursts.</p>\n<p><strong>Benefit</strong> You gain resistance 20 to the attuned energy type. Whenever you score a hit with a melee weapon, add [[/r 2d6]] damage to the attuned energy type. In addition, you can end the benefits of this mutagen to unleash a @Template[type:cone|distance:30] of energy that deals [[/r 3d6]] damage of the attuned type for every full 10 minutes of duration remaining (@Check[type:reflex|dc:32|basic:true|name:Major Energy Mutagen] save).</p>\n<p><strong>Drawback</strong> You gain weakness 5 to the other three energy types.</p>\n<p><strong>Duration</strong> 1 hour.</p>\n<p>@Compendium[pf2e.equipment-effects.Effect: Energy Mutagen (Major)]{Effect: Energy Mutagen (Major)}</p>"
```

Comment:
```suggestion
            "value": "<p><strong>Activate</strong> <span class=\"pf2-icon\">A</span> Interact</p>\n<hr />\n<p>When created, this mutagen is attuned to one of four energy types: acid, cold, electricity, or fire. When consumed, the mutagen suffuses your body with energy that spills out whenever you attack. At higher levels, it can even grant you the ability to unleash the energy in controlled bursts.</p>\n<p><strong>Benefit</strong> You gain resistance 20 to the attuned energy type. Whenever you score a hit with a melee weapon, add [[/r 2d6]] damage to the attuned energy type. In addition, you can end the benefits of this mutagen to unleash a @Template[type:cone|distance:30] of energy that deals [[/r 3d6]] damage of the attuned type for every full 10 minutes of duration remaining (@Check[type:reflex|dc:32|basic:true] save).</p>\n<p><strong>Drawback</strong> You gain weakness 5 to the other three energy types.</p>\n<p><strong>Duration</strong> 1 hour.</p>\n<p>@Compendium[pf2e.equipment-effects.Effect: Energy Mutagen (Major)]{Effect: Energy Mutagen (Major)}</p>"
```
We can ditch the name at least, but I can't even tell what changed in this since github doesn't highlight all the diffs properly.

Reply:
The original change was that the description formerly read "When created, this mutigen is..."; I simply updated that to "When created, this mutagen is...".